package.json

```diff
@@ -47,9 +49,6 @@
     "stylelint-no-browser-hacks": "^1.2.1",
     "stylelint-order": "^4.0.0",
     "webpack": "^4.41.5",
-    "webpack-cli": "^3.3.10",
```

Comment:
This should be needed right?

Reply:
Also it seems to be an old version

Reply:
https://webpack.js.org/guides/getting-started/#basic-setup

Reply:
It's not used, webpack is called through Gulp.